### ISSUE TITLE
Update the CourtListener build process

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#304]
cc: @nowherenearithaca, @jraller 

We've talked a number of times about getting something like Less or webpack in place in CourtListener and documented for developers. There are a bunch of improvements this would provide:
- CSS sucks. This could provide a better styling language.
- Right now we serve 5 CSS files and 9 JS files on the homepage. To the extent we can combine and minify all of these, the site would be faster.
- Cache-busting. This is something we should be doing so that people get the latest version of resources when loading pages, but our current set up doesn't seem to do this (I suppose this is best configured server-side, but I'm no expert).

I probably won't do this myself for some time, but once the switch to Bootstrap is done (Tuesday-ish?), I suspect it'd be fairly easy and provide some good improvements.

**Comment by voutilad** [comment]
@jraller were you working on this issue at all? Looking to spend some time on some CL related needs in the coming weeks and @mlissner suggested I look into this if you're not actively pursuing.

**Comment by mlissner** [comment]
@voutilad: You've got the all clear from @jraller. I forgot to tell you I talked to him yesterday.

**Comment by mlissner** [comment]
The work I just pushed in #569 should make this easier. We just need to decide on a technology and add it to our ansible commands.

**Comment by mlissner** [comment]
Well, I just took a stab at this using https://github.com/peterbe/django-static, but I failed miserably. Django is pretty complicated when it comes to this stuff. There are a bunch of different variables:

 - STATIC_URL
 - MEDIA_ROOT
 - STATICFILES_DIR
 - STATIC_ROOT

And then this library introduces another three similar values, and everything I did resulted in `SuspiciousFileOperation`. So....this is harder than I wish it were. If anybody wants to take another stab, I'd welcome it.

**Comment by mlissner** [comment]
I've been thinking about this the wrong way. Here's the right way:

https://medium.com/@alexford/quickly-enable-really-fast-cdn-assets-with-google-pagespeed-and-amazon-cloudfront-7717f3c088ab

Basically, you set up a bunch of assets domains, then you install an apache module from Google that does automated minification for you. Hot damn.
